consensus/src/util/mod.rs

```diff
@@ -2,7 +2,24 @@
 // Parts of the project are originally copyright © Meta Platforms, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
+use aptos_types::{
+    on_chain_config::{FeatureFlag, Features},
+    validator_txn::ValidatorTransaction,
+};
+
 pub mod db_tool;
 #[cfg(any(test, feature = "fuzzing"))]
 pub mod mock_time_service;
 pub mod time_service;
+
+pub fn is_vtxn_expected(features: &Features, vtxn: &ValidatorTransaction) -> bool {
+    match vtxn {
+        ValidatorTransaction::DummyTopic1(_) | ValidatorTransaction::DummyTopic2(_) => true,
```

Comment:
this shouldn't be true, it opens ddos attacks by validators? honestly I don't think it should even exist on non-testing path

Reply:
once the corresponding feature is enabled, DKG/JWK txn are similar?

Reply:
DKG/JWK has verification logic that only valid ones can go through? the dummy txn execution just succeeds?

Reply:
fixed